constantinCurtis/src/main/java/data/collection/task/classes/Order.java

```diff
@@ -0,0 +1,52 @@
+package data.collection.task.classes;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Order {
+    private static long idCounter = 0;
+    public long id;
+    public LocalDate date;
+    public User user;
+    public List<Item> items;
+
+    //cntrl+alt+l =arrangare de cod
+
+    public Order(User user, List<Item> items) {
+        this.id = ++idCounter;
+        this.date = LocalDate.now();
+        this.user = user;
+        this.items = items;
+    }
+
+    @Override
+    public String toString() {
+        return "Order{" +
+                "id=" + id +
+                ", date=" + date +
+                ", user=" + user +
+                ", items=" + items +
+                '}';
+    }
+
+    // creating function to filter the orders by type CLOTHES
```

Comment:
In java we are operating with methods, NO Functions any more, therefore you should replace the function word with method.

Reply:
thanks, done